api/txspool.go

```diff
@@ -35,11 +35,7 @@ func (a *API) postPoolTx(c *gin.Context) {
 	}
 	// Check that tx is valid
 	if err := a.verifyPoolL2Tx(receivedTx); err != nil {
-		retBadReq(&apiError{
-			Err:  err,
-			Code: ErrParamValidationFailedCode,
-			Type: ErrParamValidationFailedType,
-		}, c)
```

Comment:
I think, It'd be great if we could keep this way of returning errors identifying all the possible errors that can be returned